### ISSUE TITLE
Latest version has missing turnmarker.css

### ISSUE THREAD

**Issue by SilentDirge** [kckaiwei/TurnMarker-alt#114]
For Version 2.9.0, instead of a turnmarker.css there is a turnmarker.less in the css directory. This results in an error and the module not loading.

I renamed turnmarker.less to turnmarker.css and that fixed it. Everything seemed to work fine in FVTT V10 but can't confirm there are zero issues. I would create a pull request but I don't think this repo is being maintained so this is more of an FYI for anyone running into this problem.

**Comment by coffiarts** [comment]
Sounds like a nice finding... However, I ran a quick check, but it doesn't seem to work for me. Still no turn marker showing up, but tons of console errors instead.

**Comment by Cyberwel** [comment]
I came accross the same issuse as repoirted by @SilentDirge, I also found renaming the file to .css allowed the module to load in the world - A quick test shows its working

I should say i still get the error in the console on loading foundry 
![image](https://user-images.githubusercontent.com/124403329/216714801-3fd0c825-3e60-46e6-aff4-28313989faf0.png)

**Comment by jamiltron** [comment]
Probably don't want to just rename the less, you want to compile it to css. You'll need to have installed the less compiler, most likely via `npm install -g less`, then you'll go to the css directory and run `lessc turnmarker.less turnmarker.css`

**Comment by duckpuppy** [comment]
> Probably don't want to just rename the less, you want to compile it to css. You'll need to have installed the less compiler, most likely via `npm install -g less`, then you'll go to the css directory and run `lessc turnmarker.less turnmarker.css`

This is true in general for `less` files... however, in this specific case the content of the file is exactly the same as the compiled output. It only contains a single CSS rule that requires no actual compilation.

**Comment by DarkenedOrigins** [comment]
i put a pr in lets see how it goes

**Comment by illandril** [comment]
For any late v10 upgraders who come across this issue, here are two alternative modules that provide similar functionality:
- [Combat Booster](https://foundryvtt.com/packages/combatbooster)
- [Illandril's Turn Marker](https://foundryvtt.com/packages/package/illandril-turn-marker)
